### PR TITLE
Update proof fixtures

### PR DESCRIPTION
**Pull request by tplooker**
Depended on #208 being merged ahead

### CODE REVIEW

File: draft-looker-cfrg-bbs-signatures.md [hunk 1]
```diff
@@ -1259,6 +1271,156 @@ Along with the SK value as defined in (#key-pair) as inputs into the Sign operat
...
+{{ $proofFixtures.proof002.presentationMessage }}
+```
+
+And public key of the signer
+
+```
+{{ $proofFixtures.proof002.signerPublicKey }}
+```
+
+Yields the following proof
```
Comment (BasileiosKal) [inline draft-looker-cfrg-bbs-signatures.md:1362]: Maybe it's naive, but could this be confusing until we have the ability to test with a deterministic proofGen??

**Comment by christianpaquin** [comment]
Question about the values used to generate these test vectors: do all the signatures and proofs use the generators in `fixtures/generators.json`, and if so, are the two first entries `Q1` and `Q2` and the remaining ones `H1` ... `H8` ?

**Comment by tplooker** [comment]
DIscussed on WG call 20th Sept, will add a note to capture the fixtures are non-deterministic in nature.

**Comment by christianpaquin** [comment]
It would be great to also add the fixtures for the sha256 ciphersuite

**Comment by christianpaquin** [comment]
> It would be great to also add the fixtures for the sha256 ciphersuite

Any ETA for this, @tplooker, @BasileiosKal? Anxious to plug these in my implementation :)

**Comment by BasileiosKal** [comment]
Hey @christianpaquin! Sorry for the long delay. Had to take some time off. Iv now opened #221 with the updated fixtures. 

Note that the signature fixtures do also take into account #216. It's a simple update were the ciphersuite id is not appended with its length when hashing.

**Comment by tplooker** [comment]
Closing in favour of #221

**Pull request closed without merging**